fw/apm.c

```diff
@@ -1327,30 +1326,45 @@ __tfw_apm_crc32_calc(TfwMsgIter *it, TfwStr *chunk , struct sk_buff *skb_head,
 }
 
 bool
-tfw_apm_hm_srv_alive(int status, TfwStr *body, struct sk_buff *skb_head,
-		     void *apmref)
+tfw_apm_hm_srv_alive(TfwHttpResp *resp, TfwServer *srv)
```

Comment:
I don't really have anything against this shortening of the list of arguments, just wanted to comment that the logic as I understand it was that all arguments on the original long list were already available in the calling function, hence there was no need to derive each of them again.

Reply:
This is a not simple shortening.
Here we see what entities function uses - response and server, not a bunch of heterogenous parameters, where we can loose unused ones.
As a model for `tfw_apm_hm_srv_alive()` was used `tfw_http_hm_suspend()`.